libpod/kube.go

```diff
@@ -533,9 +525,10 @@ func (p *Pod) podWithContainers(ctx context.Context, containers []*Container, po
 	for _, vol := range deDupPodVolumes {
 		podVolumes = append(podVolumes, *vol)
 	}
+	podName := removeUnderscores(p.Name())
```

Comment:
Would it make more sense to replace the underscore with a dash? This would at least keep a long name readable.

Reply:
True, hyphens would make a long name more readable. But I was keeping it consistent with what we already do for container names when creating the yaml. I am fine with replacing them with hyphens if that is preferred.

Reply:
I have no real preference, if we already do this for containers and nobody complained then this looks fine to me.